tensorflow/python/saved_model/BUILD

```diff
@@ -409,6 +409,7 @@ py_library(
         "//tensorflow/python:dtypes",
         "//tensorflow/python:errors",
         "//tensorflow/python:framework_ops",
+        "//tensorflow/python:lib",
```

Comment:
Looks like this needs to be sorted (moved one down). I'll push it once current CI finishes.